packages/client/src/context/AuthContext.js

```diff
@@ -13,13 +13,26 @@ export const AuthContextProvider = ({ children }) => {
     return JSON.parse(localStorage.getItem("Token Object")) || false;
   });
 
+  const [resErrors, setResErrors] = useState(false);
+
   const login = useCallback(async ({ email, password }) => {
-    const result = await axios.post(`${backend}api/auth/login`, {
-      email,
-      password,
-    });
-    _dispatch(result.data);
-    localStorage.setItem("Token Object", JSON.stringify(result.data));
+    await axios
+      .post(`${backend}api/auth/login`, {
+        email,
+        password,
+      })
+      .then((response) => {
+        _dispatch(response.data);
+        localStorage.setItem("Token Object", JSON.stringify(response.data));
+      })
+      .catch((error) => {
+        if (error.response) {
+          console.log(error.response.data.message);
```

Comment:
This seems extraneous if we're applying it to the screen via `setResErrors`.

Reply:
Yes, sorry that was missed clean-up!